apps/cli/src/commands/boot-challenger.ts

```diff
@@ -213,135 +208,127 @@ const startListener = async (commandInstance: Vorpal.CommandInstance) => {
-                    commandInstance.log(`[${new Date().toISOString()}] Failed to handle missed assertions - ${(error as Error).message}`);
-                    await sendNotification(`Failed to handle missed assertions - ${(error as Error).message}`, commandInstance);
+                    console.log(`[${new Date().toISOString()}] Failed to handle missed assertions - ${(error as Error).message}`);
+                    await sendNotification(`Failed to handle missed assertions - ${(error as Error).message}`);
                 }
 
-                commandInstance.log(`[${new Date().toISOString()}] The challenger is now listening for assertions...`);
-                await startListener(commandInstance);
+                console.log(`[${new Date().toISOString()}] The challenger is now listening for assertions...`);
+                await startListener();
 
                 if (currentNumberOfRetries + 1 <= NUM_ASSERTION_LISTENER_RETRIES) {
                     const delayPerRetry = ASSERTION_LISTENER_RETRY_DELAYS[currentNumberOfRetries];
-                    commandInstance.log(`[${new Date().toISOString()}] Challenger restarting after ${delayPerRetry / 1000} seconds`);
-                    sendNotification(`Challenger restarting after ${delayPerRetry / 1000} seconds`, commandInstance);
-
+                    console.log(`[${new Date().toISOString()}] Challenger restarting after ${delayPerRetry / 1000} seconds`);
+                    sendNotification(`Challenger restarting after ${delayPerRetry / 1000} seconds`);
+                    
                     //Wait for delay per retry
-                    await (new Promise((resolve) => {
+                    await new Promise((resolve) => {
                         setTimeout(resolve, delayPerRetry);
-                    }))
+                    });
 
-                    commandInstance.log(`[${new Date().toISOString()}] Challenger restarting with ${NUM_ASSERTION_LISTENER_RETRIES - (currentNumberOfRetries + 1)} attempts left.`);
-                    sendNotification(`Challenger restarting with ${NUM_ASSERTION_LISTENER_RETRIES - (currentNumberOfRetries + 1)} attempts left.`, commandInstance);
+                    console.log(`[${new Date().toISOString()}] Challenger restarting with ${NUM_ASSERTION_LISTENER_RETRIES - (currentNumberOfRetries + 1)} attempts left.`);
+                    sendNotification(`Challenger restarting with ${NUM_ASSERTION_LISTENER_RETRIES - (currentNumberOfRetries + 1)} attempts left.`);
                 }
-
             }
 
-            clearInterval(assertionCheckInterval);
-            commandInstance.log(`[${new Date().toISOString()}] Challenger has stopped after ${NUM_ASSERTION_LISTENER_RETRIES} attempts.`);
-            sendNotification(`Challenger has stopped after ${NUM_ASSERTION_LISTENER_RETRIES} attempts.`, commandInstance);
+            clearInterval(assertionCheckInterval as NodeJS.Timeout);
```

Comment:
Very weird, seems like something is off in the tsconfig if this type needs to be added here.